aiida_testing/mock_code/_fixtures.py

```diff
@@ -189,9 +206,11 @@ def _get_mock_code(
                     f"Executable {executable_name} not found on PATH for code label '{label}'."
                 )
 
+        if code_executable_path == 'TO_SPECIFY':
+            code_executable_path = ""
```

Comment:
I guess in that case, the `TO_SPECIFY` string can be removed entirely from the code above

Reply:
yeh I just know this issue come up with aiida-lsmo, that when we didn't have the `.aiida-testing-config.yml` present it was trying to run `TO_SPECIFY` as if that was the executable path 😅

Reply:
done